pyiron/thermodynamics/hessian.py

```diff
@@ -18,40 +18,117 @@
 
 class HessianJob(GenericInteractive):
     def __init__(self, project, job_name):
-        super(GenericInteractive, self).__init__(project, job_name)
+        super(HessianJob, self).__init__(project, job_name)
         self.__version__ = "0.0.1"
         self.__name__ = "HessianJob"
-        self.interactive_cache = {
-            "forces": [],
-            "positions": [],
-            "energy_pot": []
-        }
+        self.server.run_mode.interactive = True
         self._force_constants = None
         self._reference_structure = None
-        self._next_positions = None
+        self._energy_pot = 0
```

Comment:
I do not understand this part.

Reply:
You need to put more info for me to understand what you don't understand.

Reply:
Why is the` interactive_cache` removed and replaced by individual variables?

Reply:
Because in principle we don't touch `interactive_cache` anymore to determine which quantities should be stored. Instead of creating individual variables I could also calculate the values each time, but to make it faster I did it this way.

Reply:
As I said for me merging this pull request without discussing it in detail breaks examples and it breaks the design we implemented for all other interactive jobs. I am open to discuss both but then we should also adjust all other jobs.

Reply:
Which design? `interactive_cache` is now not touched in any of the major codes (LAMMPS, SPHInX, VASP)

Reply:
And I find it extremely unfair to talk about breaking examples if there's no unit test at all and no DocString explaining how the functions are supposed to be used.